web/src/main/java/org/springframework/security/web/util/matcher/IpAddressMatcher.java

```diff
@@ -130,4 +130,11 @@ private InetAddress parseAddress(String address) {
 		}
 	}
 
+	@Override
+	public String toString() {
+		String hostAddress = this.requiredAddress.getHostAddress();
+		return (this.nMaskBits < 0) ? "IpAddressMatcher[" + hostAddress + "]"
```

Comment:
Will you please format this as `IpAddress [127.0.0.1]`. That is:

1. Remove the word `Matcher`, and
2. Add a space between `IpAddress` and the left bracket